### DISCUSSION TITLE
Yarle Exec Conversion not running.

### DISCUSSION THREAD

**Discussion by oldenough** [akosbalasko/yarle#236] [Q&A] [answered]
I am running Windows 10 64 bit on a Dell laptop.  I have yarle-evernote-to-md-4.0.0.Setup and worked through the questions that it asks, and having made it to the last step (5 I think), I press the start conversion button, and nothing happens; no log entries, nada.

I have tried disabling all of the anti-virus software, I have tried running it as an administrator, and have rebooted repeatedly.  Always the same results.  Any suggestions?

**Comment by akosbalasko** [comment, marked as answer]
Hi @oldenough ,
A new version of Windows has been released and downloadable from here: https://github.com/akosbalasko/yarle/releases/download/v4.0.3/yarle-evernote-to-md-4.0.3.Setup.exe

But please note that I realized that unfortunately EN changed the structure of the exported files in EN v10 and therefore it is  currently not supported, please you a legacy Evernote version to export your enex files .

**Reply by oldenough** [reply]
Your changes (and my download of the legacy Evernote) got things working beautifully and super fast.  Thank you.

Now, I'm going to have to figure out templates, because otherwise.... ouch.
